berth_reservations/tests/conftest.py

```diff
@@ -208,6 +208,33 @@ def notification_template_order_refunded():
     return notification
 
 
+@pytest.fixture
+def notification_template_offer_sent():
+    from payments.notifications import NotificationType
+
+    for value in NotificationType.values:
+        if not value.startswith("sms"):
+            notification = NotificationTemplate.objects.language("fi").create(
+                type=value,
+                subject="test offer sent subject, event: {{ offer.pk }}!",
+                body_html="<b>{{ offer.pk }} {{ accept_url }}</b>",
+                body_text="{{ offer.pk }} {{ accept_url }}",
+            )
+            notification.create_translation(
+                "en",
+                subject="test offer sent subject, event: {{ offer.pk }}!",
+                body_html="<b>{{ offer.pk }} {{ accept_url }}</b>",
+                body_text="{{ offer.pk }} {{ accept_url }}",
+            )
+
+    NotificationTemplate.objects.language("fi").create(
+        type=NotificationType.SMS_INVOICE_NOTICE.value,
+        subject="SMS offer notice",
+        body_html="Offer {{ offer.pk }} due date {{ due_date }}",
+        body_text="Offer {{ offer.pk }} due date {{ due_date }}",
+    )
+
+
```

Comment:
For this, I would prefer either (1) rename `notification_template_orders_approved -> notification_template_orders_templates` and let it handle _all_ the order-related templates, and use this fixture on the offer-related tests, or (2) do the same as with `notification_template_order_refunded` and just create the necessary templates.

Reply:
Right, I'll just create the necessary templates